yt_dlp/extractor/rinsefm.py

```diff
@@ -1,8 +1,24 @@
 from .common import InfoExtractor
-from ..utils import format_field, parse_iso8601
+from ..utils import format_field, parse_iso8601, traverse_obj
 
 
-class RinseFMIE(InfoExtractor):
+class RinseFMBaseIE(InfoExtractor):
+    @staticmethod
+    def _parse_entry(entry):
+        return {
+            **traverse_obj(entry, {
+                'id': ('id'),
+                'title': ('title'),
+                'url': ('fileUrl'),
+                'vcode': 'none',
+                'release_timestamp': ('episodeDate', {parse_iso8601}),
+            }),
+            'thumbnail': format_field(
+                entry, [('featuredImage', 0, 'filename')], 'https://rinse.imgix.net/media/%s', default=None),
+        }
```

Comment:
This would return `None` for `vcodec` instead of `'none'`.
```suggestion
        return {
            **traverse_obj(entry, {
                'id': ('id', {str}),
                'title': ('title', {str}),
                'url': ('fileUrl', {url_or_none}),
                'release_timestamp': ('episodeDate', {parse_iso8601}),
                'thumbnail': ('featuredImage', 0, 'filename', {str},
                              {lambda x: x and f'https://rinse.imgix.net/media/{x}'}),
            }),
            'vcodec': 'none',
        }
```